bin/v-update-white-label-logo

```diff
@@ -28,7 +26,7 @@ source_conf "$HESTIA/conf/hestia.conf"
 #                       Action                             #
 #----------------------------------------------------------#
 
-if [ "$reset" = "yes" ]; then
+if [ $reset = "yes" ]; then
```

Comment:
$reset needs shell check will other wise complain

Reply:
<img width="572" alt="Screenshot 2023-08-23 at 13 46 51" src="https://github.com/hestiacp/hestiacp/assets/9754650/210c4147-5a24-49c8-a473-3cf47c75ead4">

It is informative currently ..